packages/connect-query/src/index.ts

```diff
@@ -12,28 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export type {
-  ConnectQueryKey,
-  ConnectInfiniteQueryKey,
-} from "./connect-query-key.js";
-export {
-  createConnectQueryKey,
-  createConnectInfiniteQueryKey,
-} from "./connect-query-key.js";
-export { disableQuery, createProtobufSafeUpdater } from "./utils.js";
+export * from "@connectrpc/connect-query-core";
 export { useTransport, TransportProvider } from "./use-transport.js";
-export type { CreateInfiniteQueryOptions as UseInfiniteQueryOptions } from "./create-use-infinite-query-options.js";
+export type {
+  UseInfiniteQueryOptions,
+  UseSuspenseInfiniteQueryOptions,
+} from "./use-infinite-query.js";
 export {
   useInfiniteQuery,
   useSuspenseInfiniteQuery,
 } from "./use-infinite-query.js";
-export type { CreateQueryOptions as UseQueryOptions } from "./create-use-query-options.js";
+export type { UseQueryOptions, UseSuspenseQueryOptions } from "./use-query.js";
```

Comment:
Took this change to fix these names at their source instead of renaming exports.

Reply:
No longer applicable since v2 normalized these anyways.